docs/patterns/tools/use-tools/action-guide-without-agents.mdx

```diff
@@ -25,15 +25,15 @@ tool_set.execute_action(
 import { OpenAIToolSet } from "composio-core";
 
 const toolset = new OpenAIToolSet();
-const entity = toolset.client.getEntity("Jessica");
+const entity = toolset.client.getEntity("default");
 
-const result = await entity.execute(
-    "GITHUB_STAR_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER",
-    {
-        "owner": "composiohq",
-        "repo": "composio"
-    }
-);
+const result = await entity.execute({
+  actionName: "GITHUB_STAR_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER",
+  params: {
+    owner: "composiohq",
+    repo: "composio",
+  },
+});
```

Comment:
The updated JavaScript execute method format is more explicit and self-documenting. The new structure with `actionName` and `params` properties makes the API more intuitive and easier to understand. This is a good improvement! 👍